### ISSUE TITLE
facing issue using NFFT

### ISSUE THREAD

**Issue by rkumar-slim** [JuliaMath/NFFT.jl#127]
Hello, I am using julia v"1.5.2" and when I run 

using NFFT

J, N = 8, 16
k = range(-0.4, stop=0.4, length=J)  # nodes at which the NFFT is evaluated
f = randn(ComplexF64, J)             # data to be transformed
p = plan_nfft(k, N, reltol=1e-9)     # create plan
fHat = adjoint(p) * f                # calculate adjoint NFFT
y = p * fHat   

I get the following error:

ERROR: MethodError: no method matching plan_fft!(::Array{Complex{Float64},1}, ::UnitRange{Int64}; reltol=1.0e-9)

any help is highly appreciated

**Comment by tknopp** [comment]
Yes, the documentation refers to the current release. When using an older Julia release, it will in-turn install an old version of this package. Best is to install at least Julia 1.6.
